packages/@lwc/engine-core/src/framework/base-bridge-element.ts

```diff
@@ -125,6 +132,43 @@ export function HTMLBridgeElementFactory(
         };
     }
     defineProperties(HTMLBridgeElement.prototype, descriptors);
+    // observable attributes
+    HTMLBridgeElement.prototype.attributeChangedCallback = function (
```

Comment:
Why not adding `attributeChangedCallback` to `descriptors` and leverage the fact that we already call `defineProperties` line 134?

Reply:
Ok, will do!